### PR TITLE
e2e-Deduplicate Test scenarios with override service

### PR DESCRIPTION
**Pull request by itsmekumari**
@sawantpritam Please review the Override service account change.

### CODE REVIEW

File: e2e-test/src/test/features/DeduplicateErrorScenarios.feature [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+    When Select plugin: "Deduplicate" from the plugins list as: "Analytics"
+    Then Connect plugins: "GCS" and "Deduplicate" to establish connection
+    Then Navigate to the properties page of plugin: "GCS"
+    Then Replace input plugin property: "projectId" with value: "projectId"
+    Then Override Service account details if set in environment variables
+    Then Enter input plugin property: "referenceName" with value: "GCSReferenceName"
+    Then Enter input plugin property: "path" with value: "gcsSourcePath"
+    Then Select dropdown plugin property: "format" with option value: "csv"
+    Then Click plugin property: "skipHeader"
+    Then Capture the generated Output Schema
```
Comment (sawantpritam) [inline e2e-test/src/test/features/DeduplicateErrorScenarios.feature]: This step is needed only when we validate columns in preview data. Not required in this scenario.

Check in all scenarios. Add only if needed.

File: e2e-test/src/test/features/BigQueryToDeduplicate.feature [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+    Then Deploy the pipeline
+    Then Run the Pipeline in Runtime
+    Then Wait till pipeline is in running state
+    Then Open and capture logs
+    Then Verify the pipeline status is "Succeeded"
+    Then Close the pipeline logs
+    Then Validate OUT record count of deduplicate is equal to IN record count of sink
+
+  @BQ_SOURCE_DEDUPLICATE_TEST @BQ_SINK_TEST @PLUGIN-1207
+  Scenario: To verify complete flow of data extract and transfer from BQ source to BQ sink using Deduplicate Plugin with Number of Partitions
```
Comment (sawantpritam) [inline e2e-test/src/test/features/BigQueryToDeduplicate.feature]: Remove scenario with partition field.. we dont have way to properly validate partitions

**Review by sawantpritam — CHANGES_REQUESTED** [review]

File: e2e-test/src/test/features/DeduplicateErrorScenarios.feature (hunk 1 above)
Comment (itsmekumari) [reply e2e-test/src/test/features/DeduplicateErrorScenarios.feature]: Removed as not required, checked in all the scenarios.

File: e2e-test/src/test/features/BigQueryToDeduplicate.feature (hunk 2 above)
Comment (itsmekumari) [reply e2e-test/src/test/features/BigQueryToDeduplicate.feature]: Removed.

**Pull request closed without merging**